src/main/scala/beam/agentsim/agents/PersonAgent.scala

```diff
@@ -261,7 +242,22 @@ object PersonAgent {
     bodyVehicleId: Id[BeamVehicle],
     bodyVehicleTypeId: Id[BeamVehicleType]
   ): EmbodiedBeamTrip = {
-    if (trip.tripClassifier != WALK && trip.tripClassifier != WALK_TRANSIT) {
+    if (trip.tripClassifier != EMERGENCY && trip.tripClassifier != EMERGENCY) {
```

Comment:
This seems like a copy/paste error or something to that extent - as the condition is the same